src/components/Nav/LoginModal/KakaoAuth.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import kakaoLogin from './KakaoData';
```

Comment:
```suggestion
import { AUTH_TOKEN_URL, TOKEN_QUERY } from './KakaoData';
```
한번 가져오면 그대로 쓸 값들이라서 함수 본문보다 import 할 때 바로 named import 하는게 좋을 것 같아요